app/admin/photo.rb

```diff
@@ -0,0 +1,43 @@
+ActiveAdmin.register Photo do
+  config.sort_order = 'created_at_desc'
+
+  filter :name_contains
+
+  permit_params :image,
+                :name
+
+  index do
+    column :image do |photo|
+      image_tag photo.url, style: 'width: 5rem; height: 5rem'
```

Comment:
This is a perfect situation where you'd want to use Dragonfly to generate a cropping/smaller-sized image first before applying CSS (if any).  Right now, you're pulling in the unmodified, original image for every Photo record on the index.  If you have a lot of really large, high quality originals, this page is going to be a lot slower than it could be.

Instead, use `photo.thumb('150x150#').url` with whatever dimensions make sense for how you want the page to appear, then further adjust with CSS if necessary.  Here's additional information on Imagemagick transforms:  https://markevans.github.io/dragonfly/imagemagick

I'd also suggest adding a `class: 'admin-index-photo'` or something in a file like `app/assets/stylesheets/admin/photos.scss` and use that instead of inline styles.